services/src/main/java/org/keycloak/broker/saml/SAMLEndpoint.java

```diff
@@ -351,10 +351,10 @@ protected Response handleLoginResponse(String samlResponse, SAMLDocumentHolder h
                     return callback.error(relayState, Messages.IDENTITY_PROVIDER_UNEXPECTED_ERROR);
                 }
 
-                boolean assertionIsEncrypted = AssertionUtil.isAssertionEncrypted(responseType);
+                boolean assertionIsEncrypted = AssertionUtil.isAssertionsEncrypted(responseType);
```

Comment:
Currently Keycloak only uses first assertion and this PR does not change this behaviour. Why to introduce this change?